WcaOnRails/config/locales/en.yml

```diff
@@ -896,8 +896,8 @@ en:
       hide_requirements: "Hide registration requirements."
       #context: when displaying the registration opening/closing date
       registration_period:
-        label: "Registration period"
-        range_html: "From %{start_date_and_time} to %{end_date_and_time}."
+        label: "Online registration period"
+        range_html: "From %{start_date_and_time} to %{end_date_and_time}. Check in the rest of the competition details whether more registration periods and/or systems will be available (for example, on the spot registration)."
```

Comment:
The current wording feels a bit awkward to me. Can we rewrite this a bit? How about:

> Online registration will be open from <start> to <finish>. Other forms of registration may be allowed (such as registration at the door). Be sure to read through all the competition details to find out.

I'd also change the label from "Online registration period" to just "Registration period", as we're already clarifying what it means here.

Reply:
Good to me :)